src/game.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const readline = require('readline');
+
+const { calculatedBullsAndCows } = require('./calculatedBullsAndCows');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const game = (secret) => {
+  terminal.question('Enter a number of 4 different digits: ', (number) => {
+    if (number.length !== 4 || new Set(number).size !== 4 || isNaN(+number)) {
+      // eslint-disable-next-line no-console
```

Comment:
Use this one instead, at the page start. It will disable no-console for whole page
`/* eslint-disable no-console */`

Reply:
done!